### ISSUE TITLE
Question regarding Test wcs1-0-0:getcoverage_operations-getcoverage_request-grid_size-get-kvp-2

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/ets-wcs10#19]
Assertion:  When a GetCoverage request is made without a specific grid size and without a grid resolution, the server returns service exception .
Even though this Test case specifies request in GetKVP format, the test engine runs the request in Post-KVP format. Refer attached file wcs_grid_size_getkvp2.txt for your reference.

**Comment by lgoltz** [comment]
Tested with [1]. Relevant part of the log details:
```
Request d1e10401_1:

Test wcs1-0-0:getcoverage_operations-getcoverage_request-grid_size-get-kvp-2 type Mandatory default result Passed (s0082/d1e1687_1)

Assertion:  When a GetCoverage request is made without a specific grid size and without a grid resolution, the server returns service exception .
...
   Method: post
   URL: http://122.15.218.129:8081/OGC_WCS/OGCReqResDispatcher?
   Body: BbOx=88.72515106201172,25.00377655029297,92.85733032226562,27.480026245117188&CrS=EPSG:4326&ReQuEsT=GetCoverage&VeRsIoN=1.0.0&SeRvIcE=WCS&CoVeRaGe=sikkim&FoRmAt=GeoTIFF
   Response from parser ::
      <ServiceExceptionReport xmlns="http://www.opengis.net/ogc" version="">
   <ServiceException code="OperationNotSupported">error while parsing xml = Content is not allowed in prolog.</ServiceException>
   <!--Response received in [366] milliseconds--></ServiceExceptionReport>

Result: Passed
```

As described POST with KVP is used. Seems to be a bug.

[1] http://122.15.218.129:8081/OGC_WCS/OGCReqResDispatcher?service=wcs&request=GetCapabilities&version=1.0.0

**Comment by dstenger** [comment]
This issue was fixed by #26.
As soon as testing is finished, the feature branch will be merged into master branch.

**Comment by dstenger** [comment]
Fix was successfully tested and merged into master branch. It will be part of the upcoming release.
